tailwind.config.js

```diff
@@ -134,7 +134,10 @@ module.exports = {
       },
       opacity: {
         '30': 0.3
-      }
+      },
+      width: {
```

Comment:
💚 Does this override all other width styles? I know I left this comment before, so apologies if the answer is no and I didn't see it!

Reply:
Nah its fine, I think its because it's set up in `extend` rather than just in the `theme`, so it adds rather than replaces